projects/canopy/src/lib/forms/radio/docs/segment/guide.stories.mdx

```diff
@@ -0,0 +1,113 @@
+
+### Don't
+
+![dont](docs/segment/dont.png)
+
+1. **Don't** use when multiple items from a list can be selected, instead use Checkboxes
+2. **Don't** use when the preset options is a long list (more than five), use the Select component instead
+
+## Additional development details
+
+### Displaying the buttons at different breakpoints
+
+Segment buttons are displayed inline, unless given a `RadioStackBreakpoint`, which tells them at which breakpoint should they appear stacked on top of each other. The `RadioStackBreakpoint` can be added using the `stack` input (see table below).
+
+### LgRadioGroupComponent
+
+#### Inputs
+
+| Name  | Description                                                  | Type              | Default                               | Required |
+| ----- | ------------------------------------------------------------ | ----------------- | ------------------------------------- | -------- |
+| id    | HTML ID attribute, auto generated if not provided            | string            | `lg-radio-group-id-${nextUniqueId++}` | No       |
+| name  | Set the name value for all inputs in the group, auto-generated if not provided | string            | `lg-radio-group-${nextUniqueId++}`    | No       |
+| value | Set the default checked radio button, must match the value of the radio button | boolean or string | null                                  | No       |
+| focus | Set the focus on the fieldset                                | boolean           | null                                  | No       |
+| stack | Stack the segment buttons at a given breakpoint              | sm, md, lg        | false                                 | No       |
+
+### LgRadioButtonComponent
+
+#### Inputs
+
+| Name            | Description                                                  | Type    | Default                             | Required |
+| --------------- | ------------------------------------------------------------ | ------- | ----------------------------------- | -------- |
+| id              | HTML ID attribute, auto generated if not provided            | string  | `lg-radio-button-${++nextUniqueId}` | No       |
+| name            | HTML name attribute                                          | string  | null                                | Yes      |
+| value           | HTML value attribute                                         | string  | null                                | Yes      |
+| ariaDescribedBy | HTML ID for the corresponding element that describes the radio, if not provided it will use the hint field where appropriate. | boolean | null                                | No       |
+
+#### Outputs
+
+| Name | Description                             | Type         | Default | Required |
+| ---- | --------------------------------------- | ------------ | ------- | -------- |
+| blur | Event emitted on blur of a radio button | EventEmitter | n/a     | No       |
+
+------
+
+# Research on this component
```

Comment:
`Research on this component` and `Related` should have `##` not one `#` (we must have missed it)